pkgs/by-name/go/gokey/package.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+buildGoModule {
+  pname = "gokey";
+  version = "0.1.2-unstable-2023-11-16";
+  src = fetchFromGitHub {
```

Comment:
```suggestion

  src = fetchFromGitHub {
```

Reply:
while there is no officially enforced formatting almost all derivations follow some formatting rules

these PR's have pretty common spacing between src and meta blocks
https://github.com/NixOS/nixpkgs/pull/106917
https://github.com/NixOS/nixpkgs/pull/104908

Reply:
Sure but a loose esthetic convention should not waste real people's time. It seems like a personal nit. If it's important to nixpkgs, it should be a formatter running in CI auto-fixing it, if not, it should not play a role in decisions about PR merges. Also, you could have mentioned it in your initial comments or just fixed it since I allowed edits. Instead, you even took the time to seek out some of my past PR. Seems like a lot of effort for two whitespace characters. Please reconsider that mindset.

Reply:
appreciate you talking time for your contribution and your time to respond to me review.

> Sure but a loose esthetic convention should not waste real people's time

there are numerous reviews where people ask for such changes

>  It seems like a personal nit

nothing personal, I am just restating what I have observed in other reviews

> If it's important to nixpkgs, it should be a formatter running in CI auto-fixing it,

there is an ongoing effort to do just that, until then it's a bit on per reviewer bases

> Also, you could have mentioned it in your initial comments or just fixed it since I allowed edits.

I have many PR's to review, so sometimes I don't have time to provide full feedback, I understand your position though.

as per allowing edits, I personally feel like editing someone else work is even more intrusive, not all people understand implications.

> Instead, you even took the time to seek out some of my past PR

I often have to check past PR's to get a better understanding, in this case it revealed that common formatting was adhered to before, so I wasn't sure why this PR is different.

> Seems like a lot of effort for two whitespace characters.

that I can agree with, but nixpkgs is a huge project, and we all have to adhere to certain expectations.

> Please reconsider that mindset.

I am sorry you feel that way, lets hope I've clarified things a bit, and lets hope we keep making contributions to nixpkgs!